Problem2.java

```diff
@@ -0,0 +1,32 @@
+// Time Complexity : O(MN) M is row, N is col
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Y
+// Any problem you faced while coding this :N
+
+
+// Your code here along with comments explaining your approach
```

Comment:
Good Job! Keep up the good work. Please add some comments explaining your approach.